_posts/2025-08-03-columna-firmantes.md

```
@@ -0,0 +1,18 @@
+---
+layout: post
+title: Quieres saber el estado en que se encuentra una solicitud de línea de crédito.
+subtitle: Ahora sabrás quienes han firmado la solicitud.
+gh-repo: /capitalexpress/capitalexpress.github.io
+gh-badge: [star, fork, follow]
+tags: [Riesgo, Ejecutivo, Solicitud, Firmas]
+comments: false
+author: Catalina Pinilla
+---
+
+## Solicitud de línea de crédito - Firmas.
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: '.'

(MD026, no-trailing-punctuation)